package.json

```diff
@@ -11,14 +11,12 @@
   "scripts": {
     "lint": "eslint --max-warnings=0 src",
     "build": "tsc",
-    "test": "jest",
```

Comment:
Hey @suhailkakar we still need to manually update these PRs before we merge. We can make changes here but this PR removed our tests